src/h3lib/include/h3api.h.in

```diff
@@ -610,6 +610,30 @@ DECLSPEC void H3_EXPORT(getH3UnidirectionalEdgeBoundary)(H3Index edge,
                                                          GeoBoundary *gb);
 /** @} */
 
+/** @defgroup cellToVertex cellToVertex
+ * Functions for cellToVertex
+ * @{
+ */
+/** @brief Returns a single vertex for a given cell, as an H3 index */
+DECLSPEC H3Index H3_EXPORT(cellToVertex)(H3Index origin, int vertexNum);
```

Comment:
Would `getCellVertex` be a better name, since this function isn't actually mapping a sell to a single vertex?

Reply:
That's what I originally had, but I was trying to follow the naming RFC - this is similar in meaning to `cellToBoundary` or `directedEdgeToCells` I think, a transformation rather than a property lookup.

Reply:
Though I see what you're saying, it's one of many not 1:1. Hrm. Open to opinions, but `cellToVertexes` definitely makes sense and I like the parallel structure.

Reply:
What about `cellToVertices`? Should we spell it the right way or the "programmer way"? ;)

Reply:
We (I? I can't remember if this was a discussion) decided a while back to use `vertexes` consistently in the codebase. Both are technically correct (see https://www.merriam-webster.com/dictionary/vertex), and I think `vertexes` is easier to remember, use in parallel structure, better for non-native English speakers, etc.